### DISCUSSION TITLE
Pruning is glitched

### DISCUSSION THREAD

**Discussion by SoulHunter287** [eritislami/evobot#698] [Q&A] [answered]
When i do -pruning it sends "Message pruning is {result}" instead of "Message pruning is on/off" how do i fix it ?

**Comment by SoulHunter287** [comment, marked as answer]
nvm i fixed it thankyou :)

**Reply by Ben9986** [reply]
i have this same problem, how did you fix it?

**Reply by ryzyx** [reply]
i have the same problem too could u pls tell the fix
